src/duct-waf-workspace/duct-waf.service.ts

```diff
@@ -55,7 +55,7 @@ export class DuctWafWorkspaceService {
       wafMethodCode: payload.wafMethodCode,
       wafValue: payload.wafValue,
       numberOfTestRuns: payload.numberOfTestRuns,
-      numberOfTraversePointsWaf: payload.numberOfTraversePointsWAF,
+      numberOfTraversePointsWaf: Number(payload.numberOfTraversePointsWAF),
```

Comment:
@usmanahmederg Is it necessary to convert `numberOfTraversePointsWAF` value to number?